contracts/registrar/IZNSRootRegistrar.sol

```diff
@@ -134,13 +143,13 @@ interface IZNSRootRegistrar is IDistributionConfig {
     ) external;
 
     function registerRootDomain(
-        string calldata name,
-        address domainAddress,
-        string calldata tokenURI,
-        DistributionConfig calldata distributionConfig,
-        PaymentConfig calldata paymentConfig
+        RootDomainRegistrationParams calldata registration
     ) external returns (bytes32);
 
+    function registerMultipleRootDomains(
```

Comment:
Ideally it would have the same name as the singular function just with `Bulk` or similar at the end, so `registerRootDomainBulk( ... )`